packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-batcher.ts

```diff
@@ -1,116 +1,19 @@
 /* eslint-disable class-methods-use-this */
 /* eslint-disable valid-jsdoc */
 
-import {isEmpty, merge} from 'lodash';
 import Batcher from '../batcher';
-import {
-  ClientEvent,
-  MetricEventNames,
-  MediaQualityEventAudioSetupDelayPayload,
-  MediaQualityEventVideoSetupDelayPayload,
-} from '../metrics.types';
+import {prepareDiagnosticMetricItem} from './call-diagnostic-metrics.util';
```

Comment:
I would move this to call-diagnostic-metrics-batcher.util.ts

Reply:
It's being called from `call-diagnostic-metrics` and `call-diagnostic-metrics-batcher`, so I think it is fine where it is.  If you feel strongly, I will create a new `call-diagnostic-metrics-batcher.util.ts` file.